src/Common/src/TypeSystem/Common/TypeSystemHelpers.cs

```diff
@@ -294,5 +294,57 @@ public static MethodDesc InstantiateAsOpen(this MethodDesc method)
 
             return method;
         }
+
+        /// <summary>
+        /// Returns true if this is a type that doesn't require marshalling.
+        /// </summary>
+        public static bool IsBlittable(this TypeDesc type)
```

Comment:
I do not think this should be in the common type system helpers. It should stay in the simplified interop impl that we got. The complete implementation would be pretty complex and bring a bunch of interop with it.

As far as I can tell - it is just used for sanity checks, right? For sanity checks, we should just have a simple method that checks there are no GCPointers being passed to PInvoke.